### PR TITLE
Migrate Messenger demo notebook to using pds.peppi as an API client

### PR DESCRIPTION
**Pull request by tloubrieu-jpl**
## 🗒️ Summary
A lot of the code has been moved in pds.peppi to simplify the notebook.

The PR https://github.com/NASA-PDS/peppi/pull/65 is needed to make the notebook work.

## ⚙️ Test Data and/or Report

Run the notebook.

## ♻️ Related Issues

Fixes https://github.com/NASA-PDS/search-api-notebook/issues/75

### CODE REVIEW

**Comment by jordanpadams** [comment]
@tloubrieu-jpl have we reloaded their updated data? or are we still ignoring/hacking their browse references to work?

File: notebooks/img/leafmap_mess.ipynb [hunk 1]
```diff
@@ -1828,13 +219,22 @@
...
+    "#    \"stroke\": True,\n",
+    "#    \"color\": \"#0000ff\",\n",
+    "#    \"weight\": 2,\n",
+    "#    \"opacity\": 1,\n",
+    "#    \"fill\": True,\n",
+    "#    \"fillColor\": \"#0000ff\",\n",
+    "#    \"fillOpacity\": 0.1,\n",
+    "#}\n",
+    "#gdf4001_geojson = leafmap.gdf_to_geojson(gdf_4001, epsg=\"4326\")\n",
+    "#merc_map.add_geojson(gdf4001_geojson, layer_name=\"4001 Collection\", style = style1, hover_style = hover_style)\n",
```
Comment (collinss-jpl) [inline notebooks/img/leafmap_mess.ipynb:233]: Should this get removed before merge?

File: notebooks/img/leafmap_mess.ipynb [hunk 2]
```diff
@@ -31,1678 +33,67 @@
...
   {
    "cell_type": "markdown",
    "id": "1d48cd0a-373d-4428-a17b-83734f9dc095",
    "metadata": {},
    "source": [
-    "### Load the 4001 bundle and 1001 bundle\n",
+    "### Load the 4001 and 1001 collections\n",
     "\n",
-    "The function below requests the url for each product and sorts the data into a pandas dataframe for later use. "
+    "The following block extra the browse and data collections and merge them together. \n",
```
[the remark this answers is not in the document]
Comment (tloubrieu-jpl) [reply notebooks/img/leafmap_mess.ipynb:48]: Thanks, done

File: notebooks/img/leafmap_mess.ipynb [hunk 3]
```diff
@@ -31,1678 +33,67 @@
...
+    "\n",
+    "Note: by an extra-ordinary circumptances, for collection 1001, none of the ref_lid_data in the browse collection match the lid of the data product collection."
```
[the remark this answers is not in the document]
Comment (tloubrieu-jpl) [reply notebooks/img/leafmap_mess.ipynb:50]: Thanks, done

File: notebooks/img/leafmap_mess.ipynb (hunk 1 above)
Comment (tloubrieu-jpl) [reply notebooks/img/leafmap_mess.ipynb:233]: Yes, I am always a bit sentimental and afraid of throwing away code which works but you are right, that pollutes the readibility of the notebook and I removed all the useless references to the collection 4001.

**Pull request merged**